omnigibson/prims/prim_base.py

```diff
@@ -158,6 +164,22 @@ def state_size(self):
         # This is the cached value
         return self._state_size
 
+    def dump_state(self, serialized=False):
+        """
+        Dumps the state of this object in either dictionary of flattened numerical form.
+
+        Args:
+            serialized (bool): If True, will return the state of this object as a 1D numpy array. Otherewise, will return
+                a (potentially nested) dictionary of states for this object
+
+        Returns:
+            dict or n-array: Either:
+                - Keyword-mapped states of this object, or
+                - encoded + serialized, 1D numerical np.array capturing this object's state
+        """
+        assert self._initialized, "Prim must be initialized before dumping state!"
+        return self._dump_state(serialized=serialized)
```

Comment:
this should call super dump_state(), not the internal call _dump_state()

Reply:
omg yes good catch. fixed